src/assets/styles/components/_pagination.css

```
@@ -0,0 +1,55 @@
+.pagination {
+    padding-block: 2.5rem;
+    text-align: center;
+    inline-size: 75%;
+    margin-inline: auto;
+}
+
+.pagination ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    list-style: none;
+    padding-inline: 1.5rem;
+}
+
+.pagination a {
+    align-items: center;
+    block-size: 2rem;
```

@chosww the pagination needs to respond to the "enhance inputs" preference.  You can do something like the following to the various sizing declarations as needed.

```suggestion
    block-size: calc(2em * var(--fl-enhance-font-size-factor, 1));
```